src/pudl/transform/eia923.py

```diff
@@ -308,24 +308,48 @@ def _aggregate_generation_fuel_duplicates(
         'fuel_consumed_mmbtu': "sum",
         'fuel_consumed_for_electricity_mmbtu': "sum",
         'net_generation_mwh': "sum",
-        # We can safely select the first fuel_type_code_aer because we know they are the same for each group of duplicates.
-        'fuel_type_code_aer': "first"
+        # We can safely select the first fuel_type_code_* because we know they
+        # are the same for each group of duplicates.
+        'fuel_type_code_aer': "first",
+        'fuel_type_code_pudl': "first",
```

Comment:
@bendnorman the problem was that `fuel_type_code_pudl` wasn't originally included here with an aggregation function, so in the case of the duplicated records which were getting aggregated, it ended up with an NA value. This affected several hundred records.